x-pack/docs/en/security/securing-communications/node-certificates.asciidoc

```diff
@@ -75,10 +75,26 @@ signing requests (CSR) for the nodes in your cluster. For more information, see
 <<certutil>>.
 --
 
-. Copy the node certificate to the appropriate locations.
+. Optional: Generate the necessary certificates to encrypt HTTP client
```

Comment:
```suggestion
. Optional: Generate additional, specific certificates to encrypt HTTP client
```

The ones they created above can as well be used for HTTP, depending on the environment and/or use case.